strapi/strapi
# fix(content-releases): add bulk add to release to v5

### What does it do?

Add again the bulk add to release feature to Strapi v5.

### How to test it?

1. On EE edition, go the the list view of an CT with draft & publish enabled
2. Select multiple entries and try to add them to one release
3. You should see in the list view table the releases column
